config.mk

```diff
@@ -22,3 +22,6 @@ RUN_SERVER_IN_BACKGROUND ?= true
 #
 # Posible options: test or qa
 LDAP_DATA ?= test
+
+# Mock the CWS.
+MM_ENABLE_CWS_MOCK = false
```

Comment:
I'd use MM_ENABLE_CWS_MOCK ?= false so by default is false but could be overridden by passing the env. variable